File: interview_prep/Fundamentals/threesAndFives.py
# Create function ThreesFives() that adds each value from 100 and 4000000 (inclusive) if that value is evenly divisible by 3 or 5 but not both. Display the final sum in the console.

def threesFives(val1,val2, div1, div2):
    sum = 0
    valueArr = []
    for i in range(val1, val2 + 1, 1):
        if i % div1 == 0:
            if i % div2 == 0:
                continue
            else:
                sum += i
                valueArr.append(i)
                
        elif i % div2 == 0:
            sum += i
            valueArr.append(i)
        else: 
            continue
    return [sum, valueArr]
            
        
print(threesFives(3, 60, 2, 3)) 

File: interview_prep/Fundamentals/threesAndFives.js
// # Create function ThreesFives() that adds each value from 100 and 4000000 (inclusive) if that value is evenly divisible by 3 or 5 but not both. Display the final sum in the console.

function threesFives(val1, val2) {
    let sum = 0
    let valArr = []
    for (let i = val1; i < val2 + 1 ; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            if (i % 3 == 0 && i % 5 == 0) {
                continue
            }
            else {
                sum += i
                valArr.push(i)
            }
            }
            else { 
                continue
            }
        }
        return [sum, valArr]
    }

    console.log(threesFives(3, 60))



// es6

